addons/test_mail/tests/test_performance.py

```diff
@@ -1875,7 +1875,7 @@ def test_message_post_loop(self):
         self.push_to_end_point_mocked.reset_mock()  # reset as executed twice
         self.flush_tracking()
 
-        with self.assertQueryCount(employee=800):  # tm: 791
+        with self.assertQueryCount(employee=829):  # tm: 801
```

Comment:
:eyes: do we know why?

Reply:
No prefetch, as each message_post is done in loop, we cannot easily check for OOO in batch. We could maybe look to put some stuff in cache, not sure if it can help. Requests are mainly simple requests to user model (based on id). I did not look at final added requests (no time), probably worth a check post merge.